demo/README.md

```diff
@@ -1,32 +1,97 @@
-## Goal: Bring up Clock Auction Demo Application
+# Clock Auction Demo Application
 
-###  Prerequisites
+##  Prerequisites
 - It is assumed that Fabric Private Chaincode repository on your machine is installed in $FPC_PATH.
 - JSON processor, jq is installed.
+- FPC Peer Docker Image has been created as per the FPC Network
+[Instructions](../utils/docker-compose/README.md#Steps) in Step 1.
+- Relevant Fabric binaires and docker images have been downloaded as per FPC
```

Comment:
Typo in "binaries"

Reply:
thanks! should be fixed in one of the fixup commits.